src/core/BufferGeometry.js

```diff
@@ -50,6 +49,13 @@ THREE.BufferGeometry.prototype = {
 
 	},
 
+	offsets: function () {
+
+		console.warn( 'THREE.BufferGeometry: .offsets has been renamed to .drawcalls.' );
+		return this.drawcalls.slice();
```

Comment:
The idea was to let old code read from the `offsets` array, but code that wants to modify the `offsets/drawcalls` array must migrate to the new API.

With `.slice()`, this will not modify or overwrite the `drawcalls` array:
```
geometry.offsets.push({ start: ... });
or
geometry.offsets = [ { start: ... } ];
```
Maybe it's overkill.